packages/botonic-core/src/core-bot.ts

```diff
@@ -91,194 +91,160 @@ export class CoreBot {
     session,
     lastRoutePath,
   }: BotRequest): Promise<BotResponse> {
-    const output = await this.runInput({ input, session, lastRoutePath })
+    const requestCoreContext: RequestCoreContext = {
```

Comment:
I think this name is not accurate, maybe simply `BotContext`?

Reply:
applied